### PR TITLE
Create schema.txt

### PR DESCRIPTION
**Pull request by krabina**
Create a schema.txt file to automatically import Schema.org vocabulary on installation.

### CODE REVIEW

File: data/import/vocabularies/schema.txt [hunk 1]
```diff
@@ -0,0 +1,2805 @@
...
+ DigitalDocumentPermissionType|Category
+ DigitalFormat|Category
+ DigitalPlatformEnumeration|Category
+ DisabilitySupport|Category
+ DisagreeAction|Category
+ Discontinued|Category
+ DiscoverAction|Category
+ DiscussionForumPosting|Category
+ DislikeAction|Category
+ Distance|Category
```
Comment (JeroenDeDauw) [inline data/import/vocabularies/schema.txt:1759]: There also is a lowecase `distance` earlier on in the text file, of type Text. Is that correct?

**Comment by JeroenDeDauw** [comment]
That is some seriously big txt file :)

File: data/import/vocabularies/schema.txt (hunk 1 above)
Comment (krabina) [reply data/import/vocabularies/schema.txt:1759]: I guess so. [schema:Distance](https://schema.org/Distance) would be for identifying categories and [schemadistance](https://schema.org/distance) for property usage. I downloaded version 14 as CSV from schema.org

**Comment by kghbln** [comment]
I am unsure if I would like to have about 3000 properties to be imported into my blanc semantic wiki. Sure you can avoid this by changing the default configuration of SMW and preventing this from happening. However, once people realize the properties are already there.

**Comment by kghbln** [comment]
Never mind. After thinking about it I guess the stuff here just gets added to the overview in the MediaWiki namespace. In this case it is ok.

**Comment by krabina** [comment]
Excactly. It creates the file [MediaWiki:Smw_import_schema](https://www.semantic-mediawiki.org/wiki/MediaWiki:Smw_import_schema) that just sits there waiting for you to use the "imported from" property declaration. If you don't use it, nothing changes in your wiki.

**Comment by krabina** [comment]
The rationale of the PR is to make it easier for novice users to re-use vocabularies. Only three old ones are provided, so I did this update.

**Pull request merged**